### DISCUSSION TITLE
C'est quoi les Formulaires ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#27] [General]
# C'est quoi les Formulaires ? | Bonjour Angular

C'est quoi les Formulaires dans Angular ?

https://apprendre.bonjour-angular.com/cest-quoi/formulaire/

**Comment by zyrass** [comment]
Une petite typo dans le template-driven ou un attribut email seul qui n'a rien à faire ici.
